modules/va_facilities/README.v1.yml

```diff
@@ -157,7 +157,7 @@ paths:
           required: false
           schema:
             type: integer
-            default: 20
+            default: 30
```

Comment:
you changed 20 to 30 here, but nearby_facility.rb looks like it returns 20 for `per_page`

Reply:
Both `/nearby` and and `/facilities` actually do return 30 records per page by default. I haven't tracked down where that default is coming from yet though, because as you point out looking at the class leads one to believe the default is 20.